packages/salesforcedx-vscode-apex/src/commands/forceApexExecute.ts

```diff
@@ -0,0 +1,228 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import {
```

Comment:
Looks like all the code was moved to `salesforcedx-vscode-apex`. Can we delete the `forceApexExecute.ts` file from `salesforcedx-vscode-core` ?

Reply:
Ah whoops, yeah i stashed deleting it from core by accident. good catch